test/blocks/merch-card-collection/merch-card-collection.test.js

```diff
@@ -161,7 +161,23 @@ describe('Merch Cards', async () => {
 
   it('should override cards when asked to', async () => {
     const el = document.getElementById('multipleFilters');
-    el.dataset.overrides = '/override-photoshop,/override-express';
+    setConfig({
```

Comment:
I suggest putting MEP rules in own config object.
it can be a module scoped config and be exposed from `personalization.js`.

Reply:
this is what we do, in config.mep.custom

Reply:
I see 👍